Sources/FluentMongoDriver/FieldKey.swift

```diff
@@ -18,7 +18,27 @@ extension FieldKey {
 extension DatabaseQuery.Field {
     func makeMongoPath() throws -> String {
         switch self {
-        case .field(let path, _, _):
+        case .field(let field, let schema):
+        return "\(schema).\(field.makeMongoKey())"
+        case .custom:
+            throw FluentMongoError.unsupportedField
+        }
+    }
+
+    func makeProjectedMongoPath() throws -> String {
```

Comment:
@Joannis I just duplicated the methods from `DatabaseQuery.Filter.Field` but I'm not sure we need them in both places.